spec/unit/blockscore/person_spec.rb

```diff
@@ -1,10 +1,12 @@
 module BlockScore
   RSpec.describe Person do
```

Comment:
@backus I would probably recommend we change this globally across the specs. It both removes one source of possible indeterminism (RSpec modifying a class under test so it has a different interface/behaviour as in normal usage), and allows for code simplification through things like `describe_class`.